lib/DllModuleFactory.js

```diff
@@ -2,18 +2,19 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
+"use strict";
```

Comment:
Yes indeed but node v4 requires it for `let`, `const`, `class` etc. So it freaks out istanbul if not added.